### PR TITLE
add support to slice on scripts

### PR DESCRIPTION
**Pull request by felipeweb**
Add support to slice in scripts
related to https://github.com/prest/prest/issues/291
depends https://github.com/prest/template/pull/3

### CODE REVIEW

File: postgres/queries.go [hunk 1]
```diff
@@ -47,11 +47,13 @@ func (adapter *Postgres) GetScript(verb, folder, scriptName string) (script stri
 // ParseScript use values sent by users and add on script
 func (adapter *Postgres) ParseScript(scriptPath string, queryURL url.Values) (sqlQuery string, values []interface{}, err error) {
 	_, tplName := path.Split(scriptPath)
-	q := make(map[string]string)
-	pid := 1
-	for key := range queryURL {
-		q[key] = queryURL.Get(key)
-		pid++
+	q := make(map[string]interface{})
```
Comment (avelino) [inline postgres/queries.go:50]: this ideal writer test

**Review by avelino — CHANGES_REQUESTED** [review]

File: postgres/queries.go (hunk 1 above)
Comment (felipeweb) [reply postgres/queries.go:50]: @avelino  test added

**Review by avelino — APPROVED** [review]
LGTM

**Review by cassiobotaro — APPROVED** [review]
LGTM

**Pull request merged**